docs/source/user-guide/expressions/aggregation.md

```diff
@@ -134,3 +134,42 @@ This means that if you were to use a `lambda` or a custom Python function to app
 Polars will try to parallelize the computation of the aggregating functions over the groups, so it is recommended that you avoid using `lambda`s and custom Python functions as much as possible.
 Instead, try to stay within the realm of the Polars expression API.
 This is not always possible, though, so if you want to learn more about using `lambda`s you can go [the user guide section on using user-defined functions](user-defined-python-functions.md).
+
+## Behavior with empty `Series`
+
+Polars tries to follow aggregation semantics that match closely with [set theory](https://en.wikipedia.org/wiki/Empty_set#Operations_on_the_empty_set) and python semantics. This means that we might differ from SQL semantics for operations on operations on empty Series. For example, `pl.Series([], pl.Int32).sum()` is equal to `0` in Polars, where it would be a missing value or `NULL` when following SQL semantics. Consequently, `.group_by().agg()` on columns with `null` values might result in different results than would be given by an SQL engine. Below, we provide an overview of all aggregations and the return value when performed on an empty series.
```

Comment:
You used the word “semantics” 4 times in the first 3 sentences and that's quite a heavy word for a user-friendly user guide.
Here's a possible rewrite in simpler English:

When computing aggregations on empty series, Polars tries to follow [set theory](https://en.wikipedia.org/wiki/Empty_set#Operations_on_the_empty_set) and Python's behaviour.
This differs from SQL for some operations: for example, `pl.Series([], pl.Int32).sum()` is equal to 0 in Polars but it is `NULL` in SQL.
Below we provide an overview of all aggregations and the return value when performed on an empty series.